nbt.json

```diff
@@ -111,5 +111,18 @@
         "type": ["nbtSwitch",{"type":"type"}]
       }
     ]
+  ],
+  "anonymousNbt": [
```

Comment:
this is unused in prismarine-nbt

I wonder if we should include the custom type from node-minecraft-protocol